docs/2.9.0/docs/canton/usermanual/support_checklist.rst

```diff
@@ -0,0 +1,138 @@
+
+    You are able to download the native Canton logs from your production system. This is relevant as the logs are highly
+    structured and contain information that is sometimes omitted by third-party log aggregators.
+
+        * The logs can be read :ref:`using lnav<lnav>`.
+        * They include the trace-id (``tid:``)
+        * They include at least the log-level INFO.
+        * Confirm that you can upload the log files to a support ticket.
+
+    Note that you can set up Canton to create rolling log files which are included in the health dump. Both the standard
+    and the default Canton JSON logging format are fine.
+    If you need custom loggers, please add them as an extension to the built-in `logback.xml <https://github.com/digital-asset/canton/blob/main/community/app/src/main/resources/logback.xml>`_.
+
+* **Attention to WARN and ERROR logs**
+
+    You are monitoring the logs for WARN and ERROR messages. Canton logs WARN on degradations and ERROR on fatal failures.
+    There should never be an ERROR in the logs emitted by the Canton loggers.
+
+* **Finding Commands in the Logs**
+
+    You are able to find the commands that were submitted to Canton in the logs.
+    Verify that you can find the previous ping in your logs looking for ``READY-FOR-PRODUCTION`` (which
+    was the argument passed as ``id``). You should see several log entries mentioning ``Phase 1 started: Submitting commands for interpretation``,
+    one for each Daml transaction of the ping command.
+
+* **Log File Permanence**
+
+    You have a strategy in place to ensure that the logs are not lost when the node is restarted.
+
+        * Restart your node and confirm that the logs of the ping command are still available.
+
+* **Log Level Change**
+
+    You are able to change the log level of the Canton loggers using the remote console command ``node.health.set_log_level(Level.DEBUG)``.
+
+        * Confirm that the log level changes to DEBUG and change it back again.
+
+* **Command Id / Trace Id in Application Logs**
+
+    Your applications provide a trace-id or at least the command-id in their logs that can be correlated with the Canton logs.
+    Please consult our :ref:`guide on how to add end-to-end tracing <open-tracing-ledger-api-client>` to your application.
+
+        * Verify that you can find the trace-id or command-id in your application logs and correlate it with the Canton logs.
+
+.. _support_metrics:
+
+Metrics
+-------
+
+* **Access to Metrics**
+
+    You are able to extract key metrics from your production system, in particular the JVM metrics.
+
+.. _support_configuration:
+
+Configuration and Patching
+--------------------------
+
+* **Access to Configuration Files**
+
+    You are able to edit and change the configuration files of your production system if necessary.
+
+* **Patch Release Process**
+
+    You have a well-defined procedure for applying patch releases to your production system.
+
+Database Access
```

Comment:
Do you want to mention automatic backups ? It should be obvious but...

Reply:
I've added it.